pkg/apis/tenancy/v1alpha1/helper/helper.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2021 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helper
+
+import (
+	"fmt"
+	"strings"
+
+	tenancyapi "github.com/kcp-dev/kcp/pkg/apis/tenancy/v1alpha1"
+)
+
+const (
+	separator = "_"
+
+	// OrganizationCluster is the name of the logical cluster we expect to find
+	// organization inside.
+	OrganizationCluster = "admin"
+)
+
+// EncodeLogicalClusterName determines the logical cluster name for a workspace.
+// We assume that the organization that this workspace resides in is the cluster
+// it lives in.
```

Comment:
"living in" is confusing here. You want to say that we prefix the workspace name with the org cluster. It does not mean that the workspace must be on the same shard as the org workspace, does it?

Reply:
Yikes. Naming. The `Workspace` object for a workspace has a `metadata.clusterName`. This is the organizational workspace which owns the workspace. That's what this comment means.